src/layouts/ReviewRequest/components/DiffView/DiffView.tsx

```diff
@@ -0,0 +1,143 @@
+import { Box, Flex, Spacer, VStack, Text, Heading } from "@chakra-ui/react"
+import { Button, Link } from "@opengovsg/design-system-react"
+import { useEffect } from "react"
+import ReactDiffViewer, {
+  DiffMethod,
+  ReactDiffViewerProps,
+} from "react-diff-viewer"
+import { BiLeftArrowAlt } from "react-icons/bi"
+import { useParams } from "react-router-dom"
+
+import { useBlob } from "hooks/githubHooks/useBlob"
+
+import { getAxiosErrorMessage } from "utils/axios"
+
+import { useErrorToast } from "utils"
+
+export const DiffViewer = ({
+  oldValue,
+  newValue,
+  ...props
+}: ReactDiffViewerProps): JSX.Element => {
+  return (
+    <ReactDiffViewer
+      oldValue={oldValue}
+      newValue={newValue}
```

Comment:
actually rendering the image as is might not be that simple just because the image might be extremely huge (eg for their homepage); i'm not sure if there's a good solution for this beyond just showing a link to the raw image maybe? wdyt about this

Reply:
I was thinking if maybe we can set a max height/width for the image but showing a link should be ok also hmm

Reply:
actually that wouldn't work also - we would need to determine our resizing strategy; whether we want to crop, preserve aspect ratio but then maybe have whitespace or like ignore the aspect ratio but then resize (and it might look weird in that case)

Reply:
That makes sense, I think we can stick to showing a link for now.